src/tools/schemes-fragment/import-all-schemes.ipynb

```diff
@@ -0,0 +1,147 @@
+{
```

Comment:
Okay reviewing the diff on github is insane, but check out the actual file itself: 

https://github.com/microsoft/terminal/blob/dev/migrie/f/all-schemes/src/tools/schemes-fragment/import-all-schemes.ipynb